examples/connext_dds/custom_transport/c/FileTransport.c

```diff
@@ -1781,18 +1781,12 @@ RTI_INT32 NDDS_Transport_FILE_get_receive_interfaces_cEA(
     /* Set the meaningful bytes of the address */
     {
         struct in_addr ipAddr;
-        unsigned int ipAddrNetworkOrder;
 
         inet_aton(me->_property.address, &ipAddr);
-        ipAddrNetworkOrder = ipAddr.s_addr;
-        interface_array_inout[0].address.network_ordered_value[15] =
-                (ipAddrNetworkOrder) % 256;
-        interface_array_inout[0].address.network_ordered_value[14] =
-                (ipAddrNetworkOrder >> 8) % 256;
-        interface_array_inout[0].address.network_ordered_value[13] =
-                (ipAddrNetworkOrder >> 16) % 256;
-        interface_array_inout[0].address.network_ordered_value[12] =
-                (ipAddrNetworkOrder >> 24) % 256;
+        memcpy(
```

Comment:
The C/C++ linter is failing for this line. I guess the first parameter goes in the same line.

Reply:
That doesn't seem right. This is the right indentation, we'll merge it like this and I'll investigate.